docs/data/getting-started/licensing/licensing.md

```diff
@@ -41,7 +35,7 @@ Community Plan packages:
 - [`@mui/x-data-grid`](https://www.npmjs.com/package/@mui/x-data-grid)
 - [`@mui/x-date-pickers`](https://www.npmjs.com/package/@mui/x-date-pickers)
 
-### Pro Plan <span class="plan-pro"></span>
```

Comment:
Is there a particular reason that the class name is the reverse of the title?

Reply:
> Is there a particular reason that the class name is the reverse of the title?

My guess is that it was named by a French engineer 😂😛 "le plan pro"

Reply:
😁

Reply:
The objective was to have a shared `plan-` prefix we could grep in the codebase https://github.com/mui/mui-x/commit/07040919b62d0207df0ebfe585234358b491c8f6